### PR TITLE
fixed an issue on eml file with a long subject and special characters is not parsed correctly

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes:[ link to the issue](https://github.com/demisto/etc/issues/47691)

## Description
fixed an issue on eml file with a long subject and special characters is not parsed correctly.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by ShahafBenYakir — APPROVED** [review]
Nice work! What about ParseEmail v2?

File: Packs/CommonScripts/ReleaseNotes/1_6_53.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### ParseEmailFiles
+- Fixed an issue where a long subject with special characters were not parsed properly.
```
Comment (ShahafBenYakir) [inline Packs/CommonScripts/ReleaseNotes/1_6_53.md:4]:
```suggestion
- Fixed an issue where a long subject with special characters was not parsed properly.
```

File: Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py [hunk 2]
```diff
@@ -3512,7 +3512,7 @@ def unfold(s):
     :param string s: a string to unfold
     :rtype: string
     """
-    return re.sub(r'[ \t]*[\r\n][ \t\r\n]*', ' ', s).strip(' ')
+    return re.sub(r'[ \t]*[\r\n][ \t\r\n]*', ' ', s).strip(' ') if s else s
```
[the remark this answers is not in the document]
Comment (ShahafBenYakir) [reply Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py:3515]: I think you have missed this

[the remark this answers is not in the document]
Comment (moishce) [reply Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py:3515]: It can also be 'None'

[the remark this answers is not in the document]
Comment (ShahafBenYakir) [reply Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py:3515]: @moishce I think you have missed this

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**